webpack/ForemanTasks/Components/TaskDetails/Components/__tests__/__snapshots__/TaskInfo.test.js.snap

```diff
@@ -292,9 +292,7 @@ exports[`TaskInfo rendering render with Props 1`] = `
       </p>
       <p>
         A paused task represents a process that has not finished properly. Any task in paused state can lead to potential inconsistency and needs to be resolved.
-        <br />
-        The recommended approach is to investigate the error messages below and in 'errors' tab, address the primary cause of the issue and resume the task.
-        <br />
+The recommended approach is to investigate the error messages below and in 'errors' tab, address the primary cause of the issue and resume the task.
```

Comment:
Indent this please

Reply:
The snapshot is created automatically,
I guess it looks like that because the use of `\n` - new line
in the fixtures: https://github.com/theforeman/foreman-tasks/blob/9f27b147c7a13e30965616a05aa525fa3479846b/webpack/ForemanTasks/Components/TaskDetails/Components/__tests__/TaskInfo.test.js#L42

Reply:
Ah, I see. Didn't notice I was looking at a snapshot from the diff, nevermind then